public/app/features/alerting/unified/utils/config.ts

```diff
@@ -8,19 +8,19 @@ export function getAllDataSources(): Array<DataSourceInstanceSettings<DataSource
 }
 
 export function checkEvaluationIntervalGlobalLimit(alertGroupEvaluateEvery?: string) {
-  // There is a discrepancy between the duration format for evaluateEvery and unifiedAlerting.minInterval
-  // "evaluateEvery" is a Prometheus duration and supports d, w, y – "minInterval" is
-  // a strict golang time.ParseDuration compatible format
-  if (!alertGroupEvaluateEvery || !isValidPrometheusDuration(alertGroupEvaluateEvery)) {
+  // config.unifiedAlerting.minInterval should be Prometheus-compatible duration
+  // However, Go's gtime library has issues with parsing y,w,d
```

Comment:
Technically it's our own `grafana-sdk-go`'s `gtime` package :D

Reply:
Hmm, so why can't we fix it? 😄

Reply:
I think that might introduce some unwanted side-effects :D